randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -825,14 +829,10 @@ Extra - room_id: [25]
           # Core-X Requirements
           Missiles
           Combat (Beginner) or 20 DMG Missiles
-          All of the following:
-              # TODO: Heat dmg
-              # Heat DMG
-              Varia Suit
-              Any of the following:
-                  # Normal DMG
-                  Combat (Intermediate) or Normal Damage ≥ 200
-                  Combat (Beginner) and Normal Damage ≥ 100
+          Any of the following:
+              # Normal DMG
+              Combat (Intermediate) or Normal Damage ≥ 200
+              Combat (Beginner) and Normal Damage ≥ 100
```

Comment:
just quick confirmation that this room isn't heated?

Reply:
It's not heated - the red tint in the room image is the emergency alert effect